launcher/src/backend/ethereum-services/GethService.js

```diff
@@ -3,24 +3,30 @@ import { ServiceVolume } from './ServiceVolume.js'
 
 export class GethService extends NodeService {
   static buildByUserInput (network, ports, workingDir) {
+    const service = new GethService()
+    service.setId()
+    workingDir = workingDir + '-' + service.id
```

Comment:
Should be checked for trailing `/`, e. g. `workingDir = "/opt/foo/bar/"` would produce `workingDir = "/opt/foo/bar/-somekindofid"`.

How about a method to do this so it's done the same way for all services?